jaxrs/src/main/java/org/killbill/billing/jaxrs/util/JaxrsUriBuilder.java

```diff
@@ -20,27 +20,36 @@
 import java.util.Map;
 
 import javax.annotation.Nullable;
+import javax.inject.Inject;
 import javax.ws.rs.Path;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.UriBuilder;
 import javax.ws.rs.core.UriInfo;
 
 import org.killbill.billing.jaxrs.resources.JaxRsResourceBase;
 import org.killbill.billing.jaxrs.resources.JaxrsResource;
+import org.killbill.billing.util.config.JaxrsConfig;
 
 public class JaxrsUriBuilder {
 
+    private final JaxrsConfig jaxrsConfig;
+
+    @Inject
+    public JaxrsUriBuilder(JaxrsConfig jaxrsConfig) {
+        this.jaxrsConfig = jaxrsConfig;
+    }
+
     public Response buildResponse(final UriInfo uriInfo, final Class<? extends JaxrsResource> theClass,
-                                  final String getMethodName, final Object objectId, final boolean pathLikeUrl) {
-        final URI location = buildLocation(uriInfo, theClass, getMethodName, objectId, pathLikeUrl);
-        return pathLikeUrl ? Response.ok().header("Location", location.getPath()).build() : Response.created(location).build();
+                                  final String getMethodName, final Object objectId) {
+        final URI location = buildLocation(uriInfo, theClass, getMethodName, objectId);
+        return !jaxrsConfig.isJaxrsLocationFullUrl() ? Response.ok().header("Location", location.getPath()).build() : Response.created(location).build();
```

Comment:
⚠️  We want to always return '201' status `Status.CREATED`, but  `Response.ok()` will set the status to `Status.OK`. You probably need to set the status by hand calling the `status(Status.CREATED).location(.....)`